src/persistent_state.py

```diff
@@ -166,3 +168,56 @@ def delete_state(path: Iterable[str]) -> bool:
         delete_state(path)
 
     return True
+
+
+async def save_form_image_url(interaction: Interaction, image_url: str) -> bool:
+    """
+    Safely save a Google form image url to disk.
+
+    If saving the URL fails, the interaction is responded to.
+
+    Args:
+        interaction: The Nextcord interaction that triggered the image being saved.
+        image_url: The image url to save.
+
+    Returns:
+        True if the image url saved correctly, False if not.
+    """
+    try:
+        set_state(["guilds", str(interaction.guild_id), "form_image_url"], image_url)
+    except Exception as e:
+        print("Unable to set form image:", e)
+
+        await interaction.response.send_message(
+            f"Failed to upload image ({type(e)}). See the logs for more details",
```

Comment:
If the first sentence end with a period, the second one probably should also. This also applies to the other two instances.

I'm actually realizing now we're sort of inconsistent across the codebase about whether Busty's sentences end with periods, we should probably fix that

Reply:
> If the first sentence end with a period, the second one probably should also. This also applies to the other two instances.

Generally I don't agree with this. I think it's fine to have error messages without periods at the very end, but still with periods in between sentences to separate them.

With respect to the busty codebase I don't feel too strongly about whether we end our error messages with periods or not as long as we're consistent. I agree that since these error messages end up being displayed as chat messages, ending them with punctuation comes across nicely.

I've added the missing periods in 4419dc9.